### ISSUE TITLE
enhancement: animation copying

### ISSUE THREAD

**Issue by edev2020** [pixijs/spine-v8#38]
Animation copying from armature a to armature b for temporary usage.
In short armature a can play an animation from armature b if they have the same skeleton data.
This can be used to split animations in different files instead of keeping them all in a single project same way dragonbones did it:

![image](https://github.com/user-attachments/assets/69d867d5-c8ae-434f-8f61-dc92d3b453ce)

This is great for games where the character has a lot of animations and animated clothes,weapons too and you dont want to load them all but only the ones he equipped or when you dont want to share a project with all your existing animations with a freelancer but give him an empty one with the skeleton base on it and he adds the animation u commissioned him for.
In short armature a finds the animation it wants to use from armature b and copies it's json to play it inside itself.

**Comment by Zyie** [comment]
Hi

We have now moved this project to the official Esoteric Software repo for them to maintain. As a result we will be archiving this repo and closing all issues. If you still are having this issue please feel free to open an issue over on the new repo!

- [New Repo](https://github.com/EsotericSoftware/spine-runtimes)

As for migrating between the projects it should be as simple as swapping out the imports from `@pxi/spine-pixi` to `@esotericsoftware/spine-pixi-v8`
